### PR TITLE
chore: created issue forms for bug report and new features

### PR DESCRIPTION
**Pull request by ArtBlue**
## Description
Issue forms for bug report and new features

## Notes
It's a bit difficult to conceptualize this and impossible to preview. Once approved, we can merge this and give it a run. If certain things need tweaking, we can follow it up with another PR to fix them. 

If this solution works well, we'll close the other PR with issue templates.

### CODE REVIEW

**Review by LuLaValva — DISMISSED** [review]
There are a few minor grammatical errors and typos that I noticed, but before addressing those I think we can publish this to discover if it matches our expectations in terms of formatting. We can always change this on a whim, so I'm good with it going in right now.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+title: "[BUG] <title>"
+labels: ["type: bug", "Needs Triage"]
+body:
+- type: checkboxes
+  id: no-existing-verify
+  attributes:
+    label: I verified there's no existing issue for this bug.
+    description: Please ensure there's no existing issue for the bug before creating one.
+    options:
+    - label: There's no existing issues
```
Comment (LuLaValva) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:12]:
```suggestion
    - label: There are no existing issues
```
or
```suggestion
    - label: There's no existing issue
```

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  attributes:
+    label: Relevant code block
+    description: If there's a known relevant code block, paste it here.
+  validations:
+    required: false
+- type: textarea
+  id: additional-context
+  attributes:
+    label: Additional context
+    description: Links? References? Screesnhots? Anything that will give us more context about the issue you are encountering!
```
Comment (LuLaValva) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:66]:
```suggestion
    description: Links? References? Screenshots? Anything that will give us more context about the issue you are encountering!
```

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  validations:
+    required: false
+- type: markdown
```
Comment (LuLaValva) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:71]: Not sure if this last section is really necessary

File: .github/ISSUE_TEMPLATE/NEW-FEATURE-REQUEST.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+name: 🌟 New Feature request
+description: Request a new feature for Skin
+title: "[NEW FEATURE] <title>"
+labels: ["type: feature", "Needs Triage"]
+body:
+- type: textarea
+  id: feature-description
+  attributes:
+    label: Describe the feature/module in detail
```
Comment (LuLaValva) [inline .github/ISSUE_TEMPLATE/NEW-FEATURE-REQUEST.yml:9]: These labels are a little inconsistent in tone. I see these two are `Describe the feature/module in detail` and `Screenshots`. They should either be
```
Description
Screenshots
```
or
```
Describe the feature/module in detail
Share screenshots of the feature/module
```
My vote is for the shorter descriptions, but it's up to you. This also applies across the board on both of these templates.

Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/NEW-FEATURE-REQUEST.yml:9]: I'll revise this one to be consistent, but in some cases, I prefer to opt to be more descriptive than brief at the expense of losing clarity.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 3 above)
Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:71]: I'd like to see this in action before deciding. It's a nice touch IMO.

**Pull request merged**